src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
Stylesheets with a .scss extension won't work directly in HTML. You should link to a .css file instead. Make sure you compile your SCSS to CSS and link to the compiled file.